### DISCUSSION TITLE
Add a quick filter function to the collection

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#8838] [Feature Requests]
![image](https://user-images.githubusercontent.com/8030134/137543715-7d704d54-dadd-44f9-9f6a-29a8260100a0.png)

According to the conditions set by the user, a quick filter function is generated at the position above. 

For example, I have a need to select members older than 18 years old. I can add a filter condition through the configuration page of this collection, and it will be displayed as a quick filter function in the position above.

Hope to understand what I'm talking about.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
